code/modules/clothing/clothing.dm

```diff
@@ -111,6 +113,8 @@ BLIND     // can't see anything
 			flags &= ~visor_flags
 			flags_inv &= ~visor_flags_inv
 			src.mask_adjusted = 1
+			if(!hathead_only)
+				slot_flags = SLOT_HEAD | SLOT_MASK

```

Comment:
This means the head-only bandana can be worn on the mask slot, which is really weird and makes no sense.